Description: URI-escape query term
 dict.leo.org doesn't like un-escaped Umlauts
Origin: vendor
Bug-Debian: https://bugs.debian.org/951840
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-02-22

--- a/Org.pm
+++ b/Org.pm
@@ -18,6 +18,7 @@
 use MIME::Base64;
 use XML::Simple;
 use Encode;
+use URI::Encode qw(uri_encode);
 
 sub debug;
 
@@ -102,6 +103,9 @@
   my $query = $term;
   $query =~ s/\s\s*/ /g;
   $query =~ s/\s/\+/g;
+
+  # URI encode the term
+  $query = uri_encode($term);
   push @form, "search=$query";
 
   #
